packages/app/config/dependency-lint.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   allowedVersions: {
-    'ember-composable-helpers': '3.1.1 || 2.4.0'
+    'ember-composable-helpers': '3.2.0 || 2.4.0',
+    'ember-maybe-in-element': '0.4.0 || 0.2.0'
```

Comment:
Do we need both?

Reply:
Looking at these, ember dropdown element automatically uses ember-maybe-in-element 0.2.0 as a dependency so it causes an error without the conditional to have either 0.4.0 or 0.2.0